### ISSUE TITLE
How to solve dependencies between libraries 

### ISSUE THREAD

**Issue by domyown** [Alibaba-NLP/CLNER#7]
![image](https://user-images.githubusercontent.com/43026521/136683569-e3ff4782-0d36-479d-abcd-05ff86fd31e1.png)
Currently my computer has conllu(version 1.3.1 through code 'conda install -c conda-forge conllu==1.3.1') and I also tried to install allennlp(version 0.9.0) in the same way 

![image](https://user-images.githubusercontent.com/43026521/136683681-f3fa707f-001b-4f15-9dd4-66ca5b7d7c0f.png)
But allennlp which version is 0.9.0 cannot be installed. Please if you know how to solve the problem......

**Comment by wangxinyu0922** [comment]
Hi,

I suggest to use `pip install ` command instead of `conda install`. By the way, you may not need to install `allennlp` package if you do not use ELMo embeddings.

**Comment by domyown** [comment]
Thank you for quick answer! Could i ask one more question?

I pasted the code 'CUDA_VISIBLE_DEVICES=0 python train.py --config config/wnut17_doc.yaml' but it doesn't work.

![image](https://user-images.githubusercontent.com/43026521/136693563-e5fee64e-1b3f-47a7-bfc4-3137cce12772.png)

I googled it and stackoverflow said 'import matplotlib matplotlib.use('TkAgg')' would be an answer, but it doesn't work too....
